### PR TITLE
added `TINKERBELL_SKIP_NETWORKING` to `./generate_env.sh`

### PR DESCRIPTION
**Pull request by JamesPGriffith**
## Description

The allows `vagrant up provisioner` to complete.

## Why is this needed

Without this change, the provisioner VM fails to complete.

Fixes: #91 

## How Has This Been Tested?

Executing `vagrant up provisioner` with the variable as an empty string:
```
[[TRUNCATED]]
    provisioner: ++ export TINKERBELL_SKIP_NETWORKING=
    provisioner: ++ TINKERBELL_SKIP_NETWORKING=
    provisioner: + make_certs_writable
[[TRUNCATED]]
    provisioner: ++ export TINKERBELL_SKIP_NETWORKING=
    provisioner: ++ TINKERBELL_SKIP_NETWORKING=
    provisioner: + [[ -z '' ]]
    provisioner: + setup_networking ubuntu 18.04
[[TRUNCATED]]
```
Removed the `.env` file and populate the variable string in the `./generate-env.sh` and rebuilt:
```
[[TRUNCATED]
    provisioner: ++ export TINKERBELL_SKIP_NETWORKING=true
    provisioner: ++ TINKERBELL_SKIP_NETWORKING=true
[[TRUNCATED]
    provisioner: + [[ -z true ]]
    # IT WAS SKIPPED
    provisioner: + setup_osie
[[TRUNCATED]
```
This failed with running in Vagrant as expected but provides the skip behavior desired.

**Note:** I am not able to test this on bare-metal for which the option was originally created.

## How are existing users impacted? What migration steps/scripts do we need?

Existing users are likely not impacted unless the `.env` was removed and recreated after #88 was merged. I was pondering whether or not the `.generate_env.sh` file should remove and recreate the `.env` file on each execution.

## Checklist:
I have:

- [ No ] updated the documentation and/or roadmap (if required)<br>However, the new option appears to need to be added to the ['On Bare Metal with Docker'](https://docs.tinkerbell.org/setup/on-bare-metal-with-docker/) section of the documentation.
- [ No ] added unit or e2e tests<br>I honestly don't know enough Go to contribute a test for Vagrant completion.
- [ No ] provided instructions on how to upgrade<br>There's nothing to upgrade beyond a `git pull`.

### CODE REVIEW

**Review by markyjackson-taulia — APPROVED** [review]
/lgtm
Holding for another maintainer to review.

**Comment by markyjackson-taulia** [comment]
Once review is complete, please remove do not merge label

**Comment by dadrus** [comment]
Is it really the way to go? I mean the `generate_env.sh` is supposed to be used for local docker setup as well. According to my understanding, the given change might break that. What about just adding the required variable to the Vagrant file like this?
```
provisioner.vm.provision :shell,
                        path: './scripts/tinkerbell.sh',
                        env: {
                          'TINKERBELL_CONFIGURE_NAT': configure_nat,
                          'TINKERBELL_SKIP_NETWORKING': ""
                        }
```
The change is the local to the vagrant setup only.

**Pull request closed without merging**
